soc/ambiq/apollo4x/CMakeLists.txt

```diff
@@ -4,6 +4,9 @@
 #
 
 zephyr_sources(soc.c)
-zephyr_include_directories(${ZEPHYR_BASE}/soc/soc_legacy/arm/common/cortex_m)
+zephyr_include_directories(
+  .
+  ${ZEPHYR_BASE}/soc/soc_legacy/arm/common/cortex_m
```

Comment:
this folder doesn't seem to exist.

And neither does the original folder referenced here: https://github.com/zephyrproject-rtos/zephyr/blob/5b4c927c50eb032730767808b0304bf1043e2e91/soc/arm/ambiq/apollo4x/CMakeLists.txt#L7

Let's remove.

Reply:
Fixed